src/System.Runtime.Analyzers/Core/CallGCSuppressFinalizeCorrectly.cs

```diff
@@ -27,42 +29,239 @@ public abstract class CallGCSuppressFinalizeCorrectlyAnalyzer : DiagnosticAnalyz
                                                                              s_localizableMessageNotCalledWithFinalizer,
                                                                              DiagnosticCategory.Performance,
```

Comment:
Nope, the stub generator seems to have gotten it wrong. Please change it. Also can edit the System.Runtime.Analyzers.props file in this project and add a -Microsoft.Usage#CA1816 entry to it. That turns off the FxCop implementation when this rule runs.